code/modules/recycling/conveyor2.dm

```diff
@@ -315,22 +316,23 @@
 					conveyors += C
 			return
 
-/obj/machinery/conveyor_switch/oneway
-	var/convdir = 1 //Set to 1 or -1 depending on which way you want the convayor to go. (In other words keep at 1 and set the proper dir on the belts.)
-	desc = "A conveyor control switch. It appears to only go in one direction."
-
-// attack with hand, switch position
-/obj/machinery/conveyor_switch/oneway/attack_hand(mob/user)
-	if(position == 0)
-		position = convdir
-	else
-		position = 0
+	if(istype(I, /obj/item/weapon/tool/wrench))
+		if(panel_open)
+			if(oneway == 1)
+				to_chat(user, "You set the switch to two way operation.")
+				oneway = 0
+				playsound(src, I.usesound, 50, 1)
+				return
+			else
+				to_chat(user, "You set the switch to one way operation.")
+				oneway = 1
+				playsound(src, I.usesound, 50, 1)
+				return
 
-	operated = 1
-	update()
+/obj/machinery/conveyor_switch/oneway
+	oneway = 1
 
-	// find any switches with same id as this one, and set their positions to match us
-	for(var/obj/machinery/conveyor_switch/S in machines)
-		if(S.id == src.id)
-			S.position = position
-			S.update()
+/obj/machinery/conveyor_switch/examine

```

Comment:
this should be examine(), not examine.

If you fix that, the workflow complaints should hopefully disappear.